SensusUI/PromptForInputsPage.cs

```diff
@@ -202,7 +211,7 @@ public PromptForInputsPage(InputGroup inputGroup, int stepNumber, int totalSteps
 
             Button nextButton = new Button
             {
-                HorizontalOptions = LayoutOptions.FillAndExpand,
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
```

Comment:
I found the following:

previous = Start, next = End, previousNextStack = CenterAndExpand
This places the buttons immediately next to one another and centers them in the screen.

previous = next = Center, previousNextStack = CenterAndExpand
Same as above.

previous = next = previousNextStack = CenterAndExpand
Same as above.

previous = Start, next = End, previousNextStack = FillAndExpand
This places the buttons immediately next to one another and aligns them to the left of the screen.

previous = next = CenterAndExpand, previousNextStack = FillAndExpand
This centers the previous button in the left half of the screen and centers the next button in the right half.